src/EmployerAan/SFA.DAS.EmployerAan.Api/Controllers/NotificationsController.cs

```diff
@@ -29,4 +31,18 @@ public async Task<IActionResult> GetNotification([FromRoute] Guid id, [FromHeade
         };
         return Ok(result);
     }
+
+    [HttpPost]
+    [ProducesResponseType(typeof(GetNotificationResponse), StatusCodes.Status200OK)]
+    public async Task<IActionResult> CreateNotification([FromHeader(Name = Constants.ApiHeaders.RequestedByMemberIdHeader)] Guid requestedByMemberId, [FromBody] CreateNotificationModel model, CancellationToken cancellationToken)
```

Comment:
Model can be dropped and the request body can be forwarded straight to the outer api client

Reply:
Done